bridge_ui/src/utils/consts.ts

```diff
@@ -276,16 +276,16 @@ export const SOLANA_HOST = process.env.REACT_APP_SOLANA_API_URL
 
 export const ETH_RPC_HOST =
   CLUSTER === 'mainnet'
-    ? 'https://ethereum-rpc.publicnode.com'
+    ? 'https://eth-mainnet.public.blastapi.io'
```

Comment:
The full node service provided by the public node is down, and it seems that their website is also inaccessible. So I have switched the backup full node URL to blast API: https://blastapi.io/. Let me know if it's ok.